tests/sdk/flagsmith.test.ts

```diff
@@ -358,3 +358,10 @@ test('getIdentityFlags succeeds if initial fetch failed then succeeded', async (
     const flags2 = await flg.getIdentityFlags('test-user');
     expect(flags2.isFeatureEnabled('some_feature')).toBe(true);
 });
+
+test('get_user_agent_extracts_version_from_package_json', async () => {
```

Comment:
I wanted to mock the package.json to test undefined case but vi seems to not be able to do so given the way it is imported/required